MAUI/CircularProgressBar/getting-started.md

```diff
@@ -183,6 +183,4 @@ SfCircularProgressBar sfCircularProgressBar = new SfCircularProgressBar
 
 ![.NET MAUI Circular ProgressBar with customized colors](images/getting-started/style.png)
 
-N> 
-* Get the complete getting started sample from [GitHub](https://github.com/SyncfusionExamples/Getting-Started-with-.NET-MAUI-Circular-ProgressBar-control.-) link.
-* Refer to our [.NET MAUI Circular ProgressBar](https://www.syncfusion.com/maui-controls/maui-progressbar) feature tour page for its groundbreaking feature representations.
\ No newline at end of file
+N> You can refer to our [.NET MAUI ProgressBar](https://www.syncfusion.com/maui-controls/maui-progressbar) feature tour page for its groundbreaking feature representations. You can also explore our [.NET MAUI ProgressBar Example](https://github.com/syncfusion/maui-demos/tree/master/MAUI/ProgressBar) that shows you how to render the ProgressBar in .NET MAUI.
```

Comment:
N> 
* Get the complete getting started sample from [GitHub](https://github.com/SyncfusionExamples/Getting-Started-with-.NET-MAUI-Circular-ProgressBar-control.-) link.
* You can refer to our [.NET MAUI ProgressBar](https://www.syncfusion.com/maui-controls/maui-progressbar) feature tour page for its groundbreaking feature representations. You can also explore our [.NET MAUI ProgressBar Example](https://github.com/syncfusion/maui-demos/tree/master/MAUI/ProgressBar) that shows you how to render the ProgressBar in .NET MAUI.

Reply:
Done